Framework/TestHelpers/inc/MantidTestHelpers/WorkspaceCreationHelper.h

```diff
@@ -273,6 +273,12 @@ Mantid::DataObjects::EventWorkspace_sptr
 createEventWorkspaceWithFullInstrument(int numBanks, int numPixels,
                                        bool clearEvents = true);
 
+/** Create an Eventworkspace with instrument 2.0 that contains
+ * RectangularDetector's */
+Mantid::DataObjects::EventWorkspace_sptr
+createEventWorkspaceWithFullInstrument2(int numBanks, int numPixels,
```

Comment:
Perhaps there is a better name for this function that better describes what it does.